app/_data/tables/install_options.yml

```diff
@@ -32,7 +32,7 @@ features:
         enterprise: true
         support: true
         icon: /assets/images/icons/documentation/rhel.jpg
-      - name: "Amazon Linux 2"
+      - name: "Amazon Linux 2023"
```

Comment:
Don't think we want to change this. 3.4 supports 2 & 2023 according to this netlify preview.

Reply:
https://deploy-preview-5841--kongdocs.netlify.app/gateway/latest/support-policy/